app/locale/en.coffee

```diff
@@ -1539,7 +1539,33 @@
     choose_course: "Choose Course"
     choose_instructions: "Select the course from the dropdown menu you’d like to assign, then click “Assign to Selected Students.”"
     push_projects: "We recommend assigning Web Development 1 or Game Development 1 after students have finished Introduction to Computer Science! See our {{resource_hub}} for more details on those courses."
-  
+    teacher_quest_create_classroom: "Create Classroom"
+    teacher_quest_add_students: "Add Students"
+    teacher_quest_teach_methods: "Help your students learn how to `call methods`."
+    teacher_quest_teach_methods_step1: "Get 75% of at least one class through __Dungeons of Kithgard__"
+    teacher_quest_teach_methods_step2: "[Print out the Student Quick Start Guide in the Resource Hub.](http://files.codecombat.com/docs/resources/StudentQuickStartGuide.pdf)"
```

Comment:
Does this URL need to be in the translation? Seems like it could go in the view file, which I think we should prefer when possible. We still have a bunch of kinks to work out around translating text that contains links.

This looks like Markdown though; not sure how that affects that situation.